### PR TITLE
chore: Clean up stories

### PR DESCRIPTION
**Pull request by alimpens**
This PR removes unused keys, uses `args.children` instead of `render` when possible, removes unnecessary `React.Fragment`s, etc.

### CODE REVIEW

File: storybook/src/components/PageMenu/PageMenu.stories.tsx [hunk 1]
```diff
@@ -17,54 +17,49 @@ export default meta
...
         Onderzoeken
-      </PageMenu.Link>
-      <PageMenu.Link href="#" key="bezoeken">
+      </PageMenu.Link>,
+      <PageMenu.Link href="#" key={2}>
         Bezoeken
-      </PageMenu.Link>
-      <PageMenu.Link href="#" key="archiveren">
+      </PageMenu.Link>,
+      <PageMenu.Link href="#" key={3}>
```
Comment (VincentSmedinga) [inline storybook/src/components/PageMenu/PageMenu.stories.tsx:42]: I understand how numbers seem simple and nice, but keys prevent having to change other values when inserting items into a list.

**Review by VincentSmedinga — COMMENTED** [review]
Thanks! Please add a little update to `documentation/storybook.md` with the guidelines you followed here.

File: storybook/src/components/PageMenu/PageMenu.stories.tsx (hunk 1 above)
Comment (alimpens) [reply storybook/src/components/PageMenu/PageMenu.stories.tsx:42]: I'm fine either way, as long as we do it consistently. Sometimes using a string for a key and mostly using numbers implies that the string has meaning imo, which it hasn't. Always using a string would fix that, but it'd be a lot more code.

I think changing everything to strings is more work then we'd save when we have to insert an item into a list once or twice, so I'm leaning towards numbers.

Comment (VincentSmedinga) [reply storybook/src/components/PageMenu/PageMenu.stories.tsx:42]: Fair enough, I can go along with numbers.

**Review by VincentSmedinga — APPROVED** [review]

**Pull request merged**